Assets/_BForBoss/_Character/Scripts/PlayerWallRunBehaviour.cs

```diff
@@ -287,26 +379,14 @@ private float GetCameraRoll()
             }
             return Mathf.LerpAngle(cameraAngle, targetAngle, Mathf.Max(_timeSinceWallAttach, _timeSinceWallDetach) / _cameraRotateDuration);
         }
-        
-        private Vector3 CalculateWallHeadingDirection()
-        {
-            return (Vector3.Cross(ChildTransform.up, _lastWallRunNormal) * CalculateWallSideRelativeToPlayer()).normalized;
-        }
-        
-        private void LookAlongWall(Vector3 characterForward, Vector3 heading)
+
+        private Vector3 ProjectOntoWallNormalized(Vector3 direction)
         {
-            if (_hasCameraStabilized) 
-                return;
-            
-            var angleDifference = Vector3.SignedAngle(characterForward, heading, Vector3.up);
-            if (Mathf.Abs(angleDifference) < _minLookAlongWallStabilizationAngle)
-            {
-                _hasCameraStabilized = true;
-            }
-            _baseCharacter.AddYawInput(angleDifference * Time.deltaTime * _lookAlongWallRotationSpeed);
+            direction.y = 0;
+            return Vector3.ProjectOnPlane(direction, _lastWallRunNormal).normalized;
         }
 
-        private static bool GetSmallestRaycastHitIfValid(RaycastHit[] array, out RaycastHit smallest)
+        private bool GetSmallestRaycastHitIfValid(RaycastHit[] array, out RaycastHit smallest)
```

Comment:
Is there a reason this isn't static anymore?

Reply:
no, just don't see a reason to make it non static

Reply:
Isn't it generally a good idea to use static whenever you can? That was my understanding but I could be mistaken

Reply:
No not really, it's a case by case. 
https://stackoverflow.com/questions/2671496/when-to-use-static-methods

this kinda sums it up well. 
>Does it make sense to call this method, even if no object has been constructed yet?

As of right now, because it's within an instance of an object, it shouldn't be static, when you extract it out, sure. No need to build for something in the future when it's not used that way right now.